boards/arm/lpcxpresso55S69/doc/index.rst

```diff
@@ -0,0 +1,210 @@
+| PIO1_7_ | GPIO            | GREEN LED                  |
++---------+-----------------+----------------------------+
+
+System Clock
+============
+
+The LPC55S69 SoC is configured to use the internal FRO at 96MHz as a source for
+the system clock. Other sources for the system clock are provided in the SOC,
+depending on your system requirements.
+
+Serial Port
+===========
+
+The LPC55S69 SoC has 8 FLEXCOMM interfaces for serial communication.  One is
+configured as USART for the console and the remaining are not used.
+
+Programming and Debugging
+*************************
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Configuring a Debug Probe
+=========================
+
+A debug probe is used for both flashing and debugging the board. This board is
+configured by default to use the LPC-Link2 CMSIS-DAP Onboard Debug Probe,
+however the :ref:`pyocd-debug-host-tools` does not yet support this probe so you
+must reconfigure the board for one of the following debug probes instead.
+
+:ref:`lpclink2-jlink-onboard-debug-probe`
+-----------------------------------------
+
+Install the :ref:`jlink-debug-host-tools` and make sure they are in your search
+path.
+
+Follow the instructions in :ref:`lpclink2-jlink-onboard-debug-probe` to program
+the J-Link firmware. Please make sure you have the latest firmware for this
+board.
+
+:ref: `opensda-daplink-onboard-debug-probe`
+-----------------------------------------
+
+PyOCD support for this board is onging and not yet available.
+To use DAPLink's flash memory programming on this board, follow the instructions
+for `updating LPCXpresso firmware`_.
+
+Configuring a Console
+=====================
+
+Regardless of your choice in debug probe, we will use the LPC-Link2
+microcontroller as a usb-to-serial adapter for the serial console.
+
+Connect a USB cable from your PC to P6
+
+Use the following settings with your serial terminal of choice (minicom, putty,
+etc.):
+
+- Speed: 115200
+- Data: 8 bits
+- Parity: None
+- Stop bits: 1
+
+Flashing
+========
+
+Here is an example for the :ref:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: lpcxpresso55S69_cpu0
+   :goals: flash
+
+Open a serial terminal, reset the board (press the RESET button), and you should
+see the following message in the terminal:
+
+.. code-block:: console
+
+   ***** Booting Zephyr OS v1.14.0 *****
+   Hello World! lpcxpresso55S69_cpu0
+
+Debugging
+=========
+
+Here is an example for the :ref:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: lpcxpresso55S69_cpu0
+   :goals: debug
+
+Open a serial terminal, step through the application in your debugger, and you
+should see the following message in the terminal:
+
+.. code-block:: console
+
+   ***** Booting Zephyr OS zephyr-v1.14.0 *****
+   Hello World! lpcxpresso55S69_cpu0
+
+.. _LPC55S69 SoC Website:
```

Comment:
Need to also add a link for ``updating LPCXpresso firmware`` you're referencing on line 137 above.

Reply:
No need, the default is now JLink. DapLink is not yet fully working, the tutorial is upgrading the debug firmware is there to let you know you have other options, but tools are under development.

I added "This example uses the :ref:`jlink-debug-host-tools` as default. to both flashing and debugging". What do you think?